templates/addons/env/s3.yml

```diff
@@ -19,6 +16,51 @@ Mappings:
     {% endfor %}
 
 Resources:
+  {{ service.prefix }}LoggingBucket:
+    Metadata:
+      'aws:copilot:description': 'An Amazon S3 bucket to store logs from {{ service.prefix }}Bucket'
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName: !Join [-, [!FindInMap [{{ service.prefix }}BucketNameMap, !Ref Env, BucketName], logs]]
```

Comment:
`!Join [-, [!FindInMap [{{ service.prefix }}BucketNameMap, !Ref Env, BucketName], logs]]` appears twice in this template. I wonder if we can set it in a parameter to remove the risk that we might change it in one place, but not the other?